core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -496,33 +498,33 @@ public HttpHeaders additionalResponseHeaders() {
     public void setAdditionalResponseHeader(CharSequence name, Object value) {
         requireNonNull(name, "name");
         requireNonNull(value, "value");
-        updateAdditionalResponseHeaders(additionalResponseHeadersUpdater,
-                                        builder -> builder.setObject(name, value));
+        updateAdditionalHeaders(additionalResponseHeadersUpdater,
+                                builder -> builder.setObject(name, value));
     }
 
     @Override
     public void setAdditionalResponseHeaders(Iterable<? extends Entry<? extends CharSequence, ?>> headers) {
         requireNonNull(headers, "headers");
-        updateAdditionalResponseHeaders(additionalResponseHeadersUpdater,
-                                        builder -> builder.setObject(headers));
+        updateAdditionalHeaders(additionalResponseHeadersUpdater,
+                                builder -> builder.setObject(headers));
     }
 
     @Override
     public void addAdditionalResponseHeader(CharSequence name, Object value) {
         requireNonNull(name, "name");
         requireNonNull(value, "value");
-        updateAdditionalResponseHeaders(additionalResponseHeadersUpdater,
-                                        builder -> builder.addObject(name, value));
+        updateAdditionalHeaders(additionalResponseHeadersUpdater,
+                                builder -> builder.addObject(name, value));
     }
 
     @Override
     public void addAdditionalResponseHeaders(Iterable<? extends Entry<? extends CharSequence, ?>> headers) {
         requireNonNull(headers, "headers");
-        updateAdditionalResponseHeaders(additionalResponseHeadersUpdater,
-                                        builder -> builder.addObject(headers));
+        updateAdditionalHeaders(additionalResponseHeadersUpdater,
+                                builder -> builder.addObject(headers));
     }
 
-    private void updateAdditionalResponseHeaders(
+    private void updateAdditionalHeaders(
```

Comment:
Not sure why we need to rename this method. Could you explain?

Reply:
Let me revert this. Bad decision. 😆